questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java

```diff
@@ -16,8 +16,19 @@ public static void main(String[] args) {
     }
 
     private static int[] findThreeLargestNumber(int[] inputData) {
+        if (inputData == null || inputData.length == 0) {
+            return new int[]{0, 0};
```

Comment:
Default output is [-1,-1], for whatever reason - not found, null, empty

Reply:
ok, thanks I will incorporate that comment.